airgun/entities/host.py

```diff
@@ -160,6 +160,8 @@ def schedule_remote_job(self, entities_list, values, timeout=60, wait_for_result
 
     def play_ansible_roles(self, entities_list, timeout=60, wait_for_results=True):
         """Play Ansible Roles on hosts names in entities_list
+           If keyword 'All' is supplied instead of list, or no list is supplied,
```

Comment:
After review, this comment get outdated - if you don't pass `entities_list`, you will get `TypeError` about missing positional argument. Same in `delete_hosts`.